src/navigation/mainStackNavigation.js

```diff
@@ -82,7 +85,7 @@ const MainStackNavigation = ( ): React.Node => (
       <Stack.Screen
         name="StandardCamera"
         component={StandardCameraWithPermission}
-        options={hideHeader}
+        options={{ ...hideHeader, orientation: isTablet ? "all" : "portrait" }}
```

Comment:
I think this is reasonable given the info you were working with, so this is just a note to myself to make an issue to disallow rotation on non-tablets everywhere *except* the AR and Standard Cameras.

Reply:
For the react-native-orientation-locker there was an active issue with iOS and `Orientation.lockToPortrait()` so this line was also for that case but if orientation is locked for all non-tablet devices this line and `Orientation.lockToPortrait()`  may not be needed. I should have asked earlier since i was wondering if it was best to just keep orientation locked to portrait on smaller devices.

Reply:
[Per Abhas](https://inaturalist.slack.com/archives/C02G5AATVGA/p1674755429544969?thread_ts=1674699000.505279&cid=C02G5AATVGA), "I think in the AR camera & regular camera yes, but otherwise I don’t think it’s necessary", so I think we'll eventually not want to disable rotation for smaller devices, but disable it on most screens except the cameras. Again, I don't think we need to deal with that here, though.